client/Commands/EncounterCommander.ts

```diff
@@ -76,6 +76,9 @@ export class EncounterCommander {
   public DisplayDifficulty = ko.pureComputed(
     () => CurrentSettings().TrackerView.DisplayDifficulty
   );
+  public PostCombatStats = ko.pureComputed(
```

Comment:
This doesn't need to be in a computed. You might have copied this pattern from DisplayDifficulty, which is in a computed because the difficulty is always displayed (so the property is bound to the KO viewmodel in `tracker.html`)